src/resources/Microsoft.Authorization/policyDefinitions/Deny-VNET-Peer-Cross-Sub.json

```diff
@@ -9,7 +9,7 @@
     "displayName": "Deny vNet peering cross subscription.",
     "description": "This policy denies the creation of vNet Peerings outside of the same subscriptions under the assigned scope.",
     "metadata": {
-      "version": "1.0.1",
+      "version": "1.0.2",
```

Comment:
As we are adding a new parameter, we need to increment the minor version of the policy as per https://github.com/Azure/Enterprise-Scale/wiki/ALZ-Policies#versioning.
Please change policy version to "1.1.0".